doc/api/esm.md

```diff
@@ -97,7 +97,13 @@ if the nearest parent `package.json` contains `"type": "module"`.
 import './startup.js'; // Loaded as ES module because of package.json
 ```
 
-## Package Scope and File Extensions
+It is recommended that package authors always include the `"type"` field, even
+in packages where all sources are CommonJS. Being explicit about the `type` of
+the package will future-proof the package in case Node.js’ default type ever
```

Comment:
```suggestion
the package will future-proof the package if the default type ever
```

Reply:
Does it not help to clarify that the default is set by Node, and not by something else (like a theoretical `package.json` spec, or a package loader, etc.)?

Reply:
In that case, maybe this?:

```suggestion
the package will future-proof the package if the default type in Node.js ever
```

Again, I'm just trying to avoid the possessive form of `Node.js` which is absolutely a microscopic thing. Feel free to ignore if these suggestions are, in your opinion, not as good as what you have already.

Reply:
Here and elsewhere, another possibility is to drop the `'`:

```suggestion
the package will future-proof the package in case the Node.js default type ever
```